### PR TITLE
BUG: Wrong mean return by Quantile

### PR DESCRIPTION
**Pull request by luca-s**
issue #98

### CODE REVIEW

**Comment by twiecki** [comment]
thanks @luca-s!
@jameschristopher can you take a look at this?

**Comment by luca-s** [comment]
Because of issue #102 and also because we cannot assume that we have 1 period forward return for every dates in the time period we are analysing (while it is a requirement to have pricing data that spans the factor analysis time period) I changed average_cumulative_return_by_quantile to use pricing data instead of forward_returns_period_1

**Comment by twiecki** [comment]
@luca-s Is this ready for another round of review?

**Comment by luca-s** [comment]
@twiecki Yes, it is. Sorry that I kept pushing commits on top of this PR.

File: alphalens/performance.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def factor_alpha_beta(factor, forward_returns, factor_returns=None):
         t_alpha = reg_fit.tvalues[0]
         alpha, beta = reg_fit.params
 
-        alpha_beta.loc['Ann. alpha', period] = (1 + alpha) ** 252 - 1
+        alpha_beta.loc['Ann. alpha', period] = (1 + alpha) ** (252.0/period) - 1
```
Comment (twiecki) [inline alphalens/performance.py]: Good catch :+1:

File: alphalens/performance.py [hunk 2]
```diff
@@ -453,9 +453,11 @@ def average_cumulative_return_by_quantile(quantized_factor, forward_returns,
     ----------
     quantized_factor : pd.Series
         Factor quantiles indexed by date and asset.
-    forward_returns : pd.Series - MultiIndex
-        Daily forward returns indexed by date and asset and
-        optional a custom group.
+    prices : pd.DataFrame
```
Comment (twiecki) [inline alphalens/performance.py]: Why this change?

File: alphalens/plotting.py [hunk 3]
```diff
@@ -341,7 +341,7 @@ def plot_quantile_returns_bar(mean_ret_by_q,
     ax : matplotlib.Axes
         The axes that were plotted on.
     """
-
+
```
Comment (twiecki) [inline alphalens/plotting.py]: Stray tab.

File: alphalens/performance.py (hunk 2 above)
Comment (luca-s) [inline alphalens/performance.py]: I didn't want to rely on forward returns because of issue #102 and also because it's a bad assumption thinking we have 1 period forward return for every day (while it is a requirement to have daily pricing data that spans the factor analysis time period). So I changed average_cumulative_return_by_quantile to use pricing data instead of forward_returns_period_1

Comment (luca-s) [inline alphalens/performance.py]: So, I made a bad choice the first time I wrote average_cumulative_return_by_quantile and I wanted to fix it.

**Comment by twiecki** [comment]
See minor comments @luca-s, otherwise this looks great. Thanks for the contribution.

File: alphalens/performance.py (hunk 2 above)
Comment (twiecki) [inline alphalens/performance.py]: :+1:

**Comment by twiecki** [comment]
Thanks @luca-s!
